modules/deployment-model/src/main/java/org/mule/runtime/deployment/model/api/policy/PolicyTemplate.java

```diff
@@ -26,13 +26,8 @@
    */
   void dispose();
 
-  /**
-   * @return plugins deployed only inside the policy template
-   */
-  List<ArtifactPlugin> getArtifactPlugins();
```

Comment:
Can't remove this. This is API

Reply:
It's not removed (see that I also renamed getOwnArtifactPlugins to getArtifactPlugins)